tools/mk/Makefile.node.targ

```diff
@@ -29,9 +29,8 @@ else
 $(NODE_EXEC) $(NPM_EXEC) $(NODE_WAF_EXEC):
 	(mkdir -p $(BUILD) \
 		&& cd $(BUILD) \
-		&& [[ -d src-node ]] && (cd src-node && git checkout master && git pull) || git clone git://github.com/joyent/node.git src-node \
+		&& [[ -d src-node ]] && (cd src-node && git checkout master && git pull) || git clone git@github.com:nodejs/node.git --branch $(NODE_PREBUILT_VERSION) --single-branch src-node  \
 		&& cd src-node \
```

Comment:
I believe this file actually comes from [eng.git](https://github.com/joyent/eng/blob/master/tools/mk/Makefile.node.targ).  Please do not change this file to diverge from the upstream copy.

Also, for the record, it actually _says_ that [at the top of the file](https://github.com/joyent/sdc-adminui/blob/master/tools/mk/Makefile.node.targ#L15-L18).